packages-new/components/button/src/components/osds-button/styles/osds-button.size.scss

```diff
@@ -3,7 +3,9 @@
 
 .button {
   height: 3.25rem;
+  border-radius: var(--ods-size-border-radius-02);
   border-width: var(--ods-size-inset-02);
+  padding: 0 var(--ods-size-squish-06-x);
```

Comment:
why this change?
Updating default button style has impact on all other components using it, and thus should be planned carefully.